hivelvet-backend/app/src/Enum/Presets/Audio.php

```diff
@@ -32,5 +32,10 @@ class Audio extends Enum
     final public const LISTEN_ONLY                        = 'listen_only_enabled';
     final public const SKIP_ECHO_TEST                     = 'skip_echo_test';
 
-    public static string $icon = 'AudioOutlined';
+    public static string $icon                                    = 'AudioOutlined';
```

Comment:
I don't think we need the icon name here. We can find it by matching the class name.